docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/event/views-view--cgov-events--ncids_collection_condensed_past_events.html.twig

```diff
@@ -0,0 +1,16 @@
+<ul class="usa-collection usa-collection--condensed" data-dynamic-list-view="Future Events" data-dynamic-list-display="NCIDS Collection Condensed">
```

Comment:
The `data-dynamic-list-view` should probably be something other than "Future Events".